source/scss/components/_ads.scss

```diff
@@ -27,7 +26,7 @@
 
 .ad-leaderboard-wrapper {
   display: none;
-  max-height: 90px;
+  height: 90px;
```

Comment:
Vitor and I agreed this needed to be max-height so that this element didn't make an empty room when there is no Leaderboard ready to be shown on the site.

Reply:
Is there a special reason this was reverted to _height_?

Reply:
Yeah. Birgit said, it has to be that way, because it was shown otherwise. And she thinks, even if there was content, it wouldn't be shown. So maybe we should discuss this again.